lib/iris/tests/unit/fileformats/grib/save_rules/test_grid_definition_template_5.py

```diff
@@ -56,29 +56,6 @@ def test__template_number(self):
         grid_definition_template_5(self.test_cube, self.mock_grib)
         self._check_key('gridDefinitionTemplateNumber', 5)
 
-    def test__shape_of_earth_spherical(self):
-        cs = RotatedGeogCS(grid_north_pole_latitude=90.0,
-                           grid_north_pole_longitude=0.0,
-                           ellipsoid=GeogCS(52431.0))
-        test_cube = self._make_test_cube(cs=cs)
-        grid_definition_template_5(test_cube, self.mock_grib)
-        self._check_key('shapeOfTheEarth', 1)
-        self._check_key('scaleFactorOfRadiusOfSphericalEarth', 0)
-        self._check_key('scaledValueOfRadiusOfSphericalEarth', 52431.0)
-
-    def test__shape_of_earth_flattened(self):
-        ellipsoid = GeogCS(semi_major_axis=1456.0, semi_minor_axis=1123.0)
-        cs = RotatedGeogCS(grid_north_pole_latitude=90.0,
-                           grid_north_pole_longitude=0.0,
-                           ellipsoid=ellipsoid)
```

Comment:
Looks like I left it out because `RotatedGeogCS` follows the same path through the code as `GeogCS` so is independent of the tests being run here, meaning the case should already be covered by `GeogCS`.